addons/mail/static/src/components/message_action_list/message_action_list.xml

```diff
@@ -3,17 +3,17 @@
     <t t-name="mail.MessageActionList" owl="1">
         <t t-if="messageActionList">
             <div class="o_MessageActionList d-flex border rounded-sm bg-white" t-attf-class="{{ className }}" t-on-click="messageActionList.onClick" t-ref="root">
-                <i t-if="messageActionList.message.hasReactionIcon" class="o_MessageActionList_action o_MessageActionList_actionReaction fa fa-lg fa-smile-o p-2 o_cursor_pointer" t-att-title="messageActionList.addReactionText" t-on-click="messageActionList.onClickActionReaction" t-ref="actionReaction"/>
+                <i t-if="messageActionList.message.hasReactionIcon" class="o_MessageActionList_action o_MessageActionList_actionReaction fa fa-lg fa-smile-o o_cursor_pointer" t-attf-class="{{messaging.device.isSmall ? 'p-2' : 'p-1'}}" t-att-title="messageActionList.addReactionText" t-on-click="messageActionList.onClickActionReaction" t-ref="actionReaction"/>
```

Comment:
for styling, we prefer inner-whitespace with `{{ }}`, e.g. `{{  'p-2' }}` instead of `{{'p-2'}}`